internal/jujuapi/websocket_test.go

```diff
@@ -52,7 +52,7 @@ func (s *websocketSuite) SetUpTest(c *gc.C) {
 
 	mux := http.NewServeMux()
 	mux.Handle("/api", jujuapi.APIHandler(ctx, s.JIMM, s.Params))
-	mux.Handle("/model/", http.StripPrefix("/model", jujuapi.ModelHandler(ctx, s.JIMM, s.Params)))
+	mux.Handle("/model/", jujuapi.ModelHandler(ctx, s.JIMM, s.Params))
```

Comment:
I think it would be better to revert this

Reply:
see https://github.com/canonical/jimm/pull/1357#discussion_r1751582926